src/jit/lsra.cpp

```diff
@@ -8270,7 +8270,7 @@ void LinearScan::resolveRegisters()
             {
                 JITDUMP(" internal");
                 GenTreePtr indNode = nullptr;
-                if (treeNode->OperIsIndir())
+                if (treeNode->OperGet() == GT_IND)
```

Comment:
It is my belief that this code is obsolete. It is certainly not required for `GT_NULLCHECK`, for which the address is never contained. For `GT_STORE_IND`, I could not find any use of internal registers in codegen.